Example/README.md

```diff
@@ -6,6 +6,19 @@
 
 Donut Counter is a sample application for building with the [Square Mobile Payments SDK](https://developer.squareup.com/docs/mobile-payments-sdk) on iOS. The app demonstrates how to install and initialize the SDK in a SwiftUI project as well as utilizing the APIs and user interfaces provided by the SDK to authorize a merchant, pair and manage card readers, and take a payment. To get started, follow the steps below.
 
+## SDK Quick Reference
+
+| Reference                        | Sample App Project Location                   |
+| -------------------------------- | ----------------------------------------- |
+| 📝 Configuring the SDK           | [Shared/Config.swift](./Shared/Config.swift) |
```

Comment:
Config.swift is not super useful, it only for account. does not really configure the sdk

Reply:
Makes sense since it's not related to SDK. Updated 🙏